docs/installation.md

```diff
@@ -12,16 +12,44 @@ Docker Compose. Now install the dependencies and start the containers. Docker wi
 starts the containers. This may take a while ...
 
 ```bash
-$ docker run --rm -it --volume $(pwd):/app composer/composer:master install --no-dev -o --prefer-dist --ignore-platform-reqs
+$ docker run --rm -it --volume $(pwd):/app prooph/composer:7.0 install --no-dev -o --prefer-dist --ignore-platform-reqs
 $ docker-compose up -d
 ```
 
-Call the MySQL setup script with:
+If you want to run (or write) unit tests you need to install the dev dependencies as well:
+
+```bash
+$ docker run --rm -it --volume $(pwd):/app prooph/composer:7.0 install -o --prefer-dist --ignore-platform-reqs
+```
+
+### Step 2 - Install event store adapter
+
+prooph offers two database adapters for `prooph/event-store` (at the moment).
+Pick the one you want to play with and install it via composer.
+
+#### Doctrine DBAL Adapter
+Install:
+
+`docker run --rm -it --volume $(pwd):/app prooph/composer:7.0 require prooph/event-store-doctrine-adapter`
+
+Configure the MySQL setup script with:
 
 ```bash
 $ docker-compose run --rm php sh bin/setup_mysql.sh
 ```
 
+#### MongoDB Adapter
+Install:
+
+`docker run --rm -it --volume $(pwd):/app prooph/composer:7.0 require prooph/event-store-mongodb-adapter`
```

Comment:
As I said, the current mongodb adapter DOES NOT WORK with php 7. This is because the used mongo-extension has no php 7 support. There is a plan to upgrade to use the new mongodb-extension which has php 7 support.